special-pages/pages/new-tab/app/favorites/components/Tile.module.css

```diff
@@ -33,6 +31,7 @@
     justify-items: center;
     width: 4rem;
     height: 4rem;
+    margin-bottom: 4px;
```

Comment:
This is here as an alternative to gap, as I saw issues in this particular case in BigSur